### DISCUSSION TITLE
An invitation to a General Meeting  - 2nd November, 2024!

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#19] [Announcements]
Dear Team Mates! @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-marketing-comms @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-hr @clentafrica/wg-sustainability-esg @clentafrica/wg-marketing-comms @clentafrica/wg-community-managers @clentafrica/wg-partnerships @clentafrica/wg-programs @clentafrica/wg-hr @clentafrica/wg-health @clentafrica/wg-design @clentafrica/wg-energy @Onodjayeke 

I hope you’re all doing well!

I am thrilled to announce a general meeting for CLENT Africa scheduled for Saturday, November 2, 2024, at 4 PM UTC. This meeting is open to everyone—from design and marketing to people operations, and even if you’re still finding your way around joining a team!

Come prepared to share ideas, collaborate, and have your key questions addressed for the benefit of all teams at CLENT Africa.

Please confirm your local time zones [here ](https://www.worldtimebuddy.com/) and let us know if the proposed date and time is convenient for you.

P.S.: All team heads are encouraged to revisit the meeting notes [here](https://docs.google.com/document/d/1s8N5hq8TyYCmcc4EPkbjx4hqfBpVRFnU8wL3PVJGX7M/edit?usp=sharing), and ensure they align with the schedules, as discussed in the previous team leads meeting especially as it has to do with individual team meetings.

I look forward to seeing you all soon!

Best Regards,
Racheal
Human Resource Team Lead

**Comment by EmmanetD** [comment]
Dear Team mates! @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-marketing-comms @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-hr @clentafrica/wg-sustainability-esg @clentafrica/wg-marketing-comms @clentafrica/wg-community-managers @clentafrica/wg-partnerships @clentafrica/wg-programs @clentafrica/wg-hr @clentafrica/wg-health @clentafrica/wg-design @clentafrica/wg-energy @Onodjayeke

Let's kindly acknowledge this invitation, thank you!

**Comment by miakolade** [comment]
Hello,

This is well received.

Thank you very much.

Regards.

**Comment by SatarDaud** [comment]
Thank you for letting us know!

**Comment by erioc** [comment]
Is the meeting on, who has the link to the meeting

**Reply by Rayccheal** [reply]
Hi @erioc, the meeting starts at 4Pm UTC.

I will send in the link now

**Comment by Obutorbeatrice** [comment]
Please I need link to the meeting

**Reply by Rayccheal** [reply]
Hi @Obutorbeatrice , meeting starts by 4PM UTC. 

I will send in the link as soon as it starts.

Thank you!

**Comment by Rayccheal** [comment]
Meeting starts in 40 minutes everyone!

I will send in the link as soon as we begin.

Thank you!!

**Comment by SatarDaud** [comment]
I am ready. Please the link to the zoom meeting

**Comment by Rayccheal** [comment]
You can join the meeting now https://us05web.zoom.us/j/81334752189?pwd=4cBva0jEhwawKgAslacvJNU8CdFW4h.1

**Comment by SatarDaud** [comment]
Okay
